### PR TITLE
Keep in sync redundant fields found in the new Social Media configlet

### LINKED ISSUE #100: Duplicated registry records for the same feature on Plone 5
Records for Twitter and Facebook are duplicated and that should lead to issues:

selection_019

selection_018

### PR DESCRIPTION
**Pull request by hvelarde**
Changes on `facebook_app_id`, `facebook_username`, or `twitter_username` fields will be reflected in both configlets.

fixes #100

### CODE REVIEW

**Comment by hvelarde** [comment]
@fredvd to avoid further issues I'll ask you to review changes on this package.

File: sc/social/like/subscribers.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        # Plone record modified; synchronize sc.social.like record
+        record = ISocialLikeSettings.__identifier__ + '.' + field
+    else:
+        logger.debug(u'Schema not being tracked')
+        return
+
+    registry = getUtility(IRegistry)
+    # we're going to modify a record and this will fire the same event
+    # we need to avoid RuntimeError due to infinite recursion
+    _FLAG = True
```
Comment (rodfersou) [inline sc/social/like/subscribers.py]: This way we have a minimum chance to have a problem if there is a new event fired in the same time.

How about to check recursion instead http://stackoverflow.com/questions/7900345/can-a-python-method-check-if-it-has-been-called-from-within-itself

Comment (rodfersou) [inline sc/social/like/subscribers.py]: I like the last answer in this S.O. http://stackoverflow.com/a/7900472

Comment (hvelarde) [inline sc/social/like/subscribers.py]: I like the solutions that uses the traceback module; I think is probably safer in the long term; I will refactor it.

File: sc/social/like/subscribers.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    registry = getUtility(IRegistry)
+    # this will fire the aditional IRecordModifiedEvent
+    try:
+        registry[record] = str(event.record.value)
+    except WrongType:
+        # Plone 5 declares records as TextLine
+        registry[record] = unicode(event.record.value)
+
+    logger.debug('{0} was synchronized; new value is "{1}"'.format(
+        repr(registry.records[record]), event.record.value))
```
Comment (idgserpro) [inline sc/social/like/subscribers.py]: I think the user should be notified about this change in the interface. And this has to be documented in README.

Comment (hvelarde) [inline sc/social/like/subscribers.py]: I agree it has to be documented, but I see no point on logging that on the event log.

Comment (idgserpro) [inline sc/social/like/subscribers.py]: We're talking about showing a message using https://docs.plone.org/develop/plone.api/docs/portal.html#portal-show-message-example to the user in the Interface.

Comment (hvelarde) [inline sc/social/like/subscribers.py]: mmm, that could make sense.

Comment (hvelarde) [inline sc/social/like/subscribers.py]: I have added some documentation and I think that's good enough.

**Pull request merged**